parsers/AR.py

```diff
@@ -143,8 +188,52 @@ def fetch_exchange(
     logger: Logger = getLogger(__name__),
 ) -> dict:
     """Requests the last known power exchange (in MW) between two zones."""
+    if target_datetime:
+        raise NotImplementedError("This parser is not yet able to parse past dates")
+
+    sorted_zone_keys = sorted([zone_key1, zone_key2])
+    sorted_codes = "->".join(sorted_zone_keys)
+    flow: Union[float, None] = None
```

Comment:
```suggestion
    sorted_zone_keys = sorted([zone_key1, zone_key2])
    sorted_codes = "->".join(sorted_zone_keys)
    flow: Optional[float] = None
    returned_datetime: datetime
```
Let's use optional instead + adding returned_datetime here so it's never unbound later.